etc/adminhtml/system/adyen_pos_cloud.xml

```diff
@@ -78,6 +78,28 @@
             <config_path>payment/adyen_pos_cloud/recurring_type</config_path>
             <comment>If you want to store the credentials for ONECLICK or RECURRING or combination of these two you need to contact support@adyen.com and ask if they can enable to store cards on company level. Changing this setting without doing this will lead to failed payments!</comment>
         </field>
+        <field id="enable_installments" translate="label" type="select" sortOrder="219" showInDefault="1"
```

Comment:
Do we need this field, can't we just apply instalments if there are configured?

Reply:
We could but this is a nice feature just to enable and disable them without losing the instalments configuration